### ISSUE TITLE
Empty object destructuring fails with object-curly-even-spacing

### ISSUE THREAD

**Issue by danez** [xjamundx/eslint-plugin-standard#9]
When using the rule `object-curly-even-spacing` and the following code
```
const foo = ({}) => a
```

The rule fails with

```
/Users/shu/code/debug-eslint/node_modules/eslint/lib/token-store.js:90
        return tokens[starts[node.range[0]] - (skip || 0) - 1];
                                 ^

TypeError: Cannot read property 'range' of undefined
    at SourceCode.api.getTokenBefore (/Users/shu/code/debug-eslint/node_modules/eslint/lib/token-store.js:90:34)
    at EventEmitter.api.(anonymous function) [as getTokenBefore] (/Users/shu/code/debug-eslint/node_modules/eslint/lib/eslint.js:888:48)
    at RuleContext.(anonymous function) [as getTokenBefore] (/Users/shu/code/debug-eslint/node_modules/eslint/lib/rule-context.js:108:33)
    at EventEmitter.ObjectPattern (/Users/shu/code/debug-eslint/node_modules/eslint-plugin-standard/rules/object-curly-even-spacing.js:180:27)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.enterNode (/Users/shu/code/debug-eslint/node_modules/eslint/lib/util/node-event-generator.js:42:22)
    at CommentEventGenerator.enterNode (/Users/shu/code/debug-eslint/node_modules/eslint/lib/util/comment-event-generator.js:98:23)
    at Controller.controller.traverse.enter (/Users/shu/code/debug-eslint/node_modules/eslint/lib/eslint.js:767:36)
    at Controller.__execute (/Users/shu/code/debug-eslint/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
```

The rule seems to assume that the object node always has properties.

originally reported here: babel/babel-eslint#238

**Comment by xjamundx** [comment]
Good catch, are you able to submit a fix?

**Comment by danez** [comment]
Seems that was already fixed in #7 and i had an old version.
